ci/test.sh

```diff
@@ -1,4 +1,90 @@
-#!/bin/bash -ex
+#!/bin/bash -x
+
+# Test runner for Ansible Conjur Collection
+
+# Test subdirectors containing a `test.sh` file
+test_directories=("conjur")
+
+# Roles containing a test subdirectory
+role_directories=("conjur_host_identity")
```

Comment:
I moved them to the role subdirectory to avoid needing to traverse upwards from the test directories, then down into the role directories. We do need to modify much of our test framework to meet ansible standards and make use of the Molecule framework as well, which may be better suited for a follow-up issue.

Reply:
> to avoid needing to traverse upwards from the test directories, then down into the role directories.

I'm not sure I follow how this would have been a requirement; but it sounds like filing a follow-up issue for cleaning up tests is the right idea. please link to it here once you have it.

Reply:
My apologies, I was unclear. For the docker-compose scripts to work as-is, we would have to use `../../`, which can be difficult to maintain, so for the time being I grouped the tests with the role. Once we centralize our images to a single build process, we shouldn't need to do this and could move around the test directories much easier.